backend/compact-connect/lambdas/nodejs/lib/report-emailer.ts

```diff
@@ -18,6 +18,8 @@ interface ReportEmailerProperties {
     sesClient: SESClient;
 }
 
+const EMAIL_IMAGE_BASE_URL = 'https://app.compactconnect.org/img/email';
```

Comment:
Let's feed the local environment's UI URL in via environment variable, rather than using production everywhere.